src/ontology/mondo-edit.obo

```diff
@@ -584370,6 +584369,20 @@ is_a: MONDO:1030005 {source="PMID:31120550"} ! mosaic SMO syndrome
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7768" xsd:anyURI
 
+[Term]
+id: MONDO:1030007
+name: hypertensive urgency
+def: "Hypertensive urgency occurs when blood pressure spikes -- blood pressure readings are 180/110 mmHg or higher -- but there is no damage to the body's organs (such as pulmonary edema, cardiac ischemia, neurologic deficits, or acute renal failure). Blood pressure can be brought down safely within a few hours with blood pressure medication." [icd11.foundation:896211058]
+synonym: "hypertensive crisis without acute organ damage" EXACT [icd11.foundation:896211058]
+synonym: "severely elevated blood pressure without acute organ damage" EXACT [icd11.foundation:896211058]
+xref: icd11.foundation:896211058 {source="MONDO:equivalentTo"}
+xref: NCIT:C197914 {source="MONDO:equivalentTo"}
+xref: SCTID:443482000 {source="MONDO:equivalentTo"}
+xref: UMLS:C0745138 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005044 {source="icd11.foundation:896211058"} ! hypertensive disorder
```

Comment:
Adding the orcid of the requester as x-ref would be good, but it doesn't look like we have it.

Reply:
I found the ORCID in another ticket from this submitter and added this